packages/components/addon/components/hds/pagination/nav/index.hbs

```diff
@@ -18,6 +18,22 @@
     </ul>
   {{/if}}
 
+  {{#if (eq this.type "truncated")}}
+    <ul class="hds-pagination-nav__page-list">
+      {{#each this.truncatedPages as |page|}}
```

Comment:
[issue] you don't need a double loop, and a double getter (`this.pages` vs `this.truncatedPages`).

Instead you can do something like this:
```
  get pages() {
    let pages = [];

    for (let i = 1; i <= this.totalPages; i++) {
      pages.push(i);
    }

    if (this.type === 'truncated') {
      return this.elliptize({ pages, current: this.currentPage });
    } else {
      return pages;
    }
  }
```